api/subscriptions/subscriptions.go

```diff
@@ -278,10 +279,16 @@ func (s *Subscriptions) setupConn(w http.ResponseWriter, req *http.Request) (*we
 		}
 	}()
 
-	return conn, closed, nil
+	paths := strings.Split(req.URL.Path, "/")
+	subject := "unknown"
+	if len(paths) > 2 {
```

Comment:
Can you add in a comment the format of the `req.URL.Path` we expect to receive? Also with an example, so to better understand what that third place element could be

Reply:
Pushed that change